sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/stream/DataStreams.java

```diff
@@ -181,12 +181,14 @@ public DataStreamDecoder(Coder<T> coder, PrefetchableIterator<ByteString> inputS
       this.inbound = new Inbound();
     }
 
-    public void seekToNextByteString() {
```

Comment:
Is there any concern with changing a public API? Or is this effectively `@Internal` since it's in fn-execution?

Reply:
Its internal.

Also hasn't been part of a release yet either since it was submitted only a few days ago.